paparazzi/build.gradle

```diff
@@ -53,6 +53,10 @@ def generateTestConfig = tasks.register("generateTestConfig") {
       writer.writeLine("app.cash.paparazzi")
       writer.writeLine(".")
       writer.writeLine("28")
+      def androidSdkFolder = System.getenv("ANDROID_SDK_ROOT")
+              ?: System.getenv("ANDROID_HOME")
+              ?: "$userHome/Library/Android/sdk"
+      writer.writeLine("${androidSdkFolder}/platforms/android-28/")
     }
   }
 }
```

Comment:
to setup, we can use a test setup similar to what the plugin would do:
```
tasks.withType(Test) { task ->
  task.dependsOn(generateTestConfig)
  task.systemProperty(
      "paparazzi.test.resources",
      generateTestConfig.map { it.outputs.files.singleFile }.get().path
  )
}
```

Reply:
I'm trying to figure a way to completely remove this from the gradle build file and have it contained in the test class itself. I'll do that in a separate PR, if that's fine.